apps/exploits/api.py

```diff
@@ -344,6 +347,46 @@ def get(self, request):
         )
 
 
+class ExploitsReportData(APIView):
+    """Export only the data required to generate the exploits report
+    TODO: Paginate
+    TODO: Check query efficiency
+    """
+
+    @extend_schema(
+        responses={
+            200: {
+                "type": "object",
+                "properties": {
+                    "exploits": {"type": "array", "items": {"type": "object"}},
+                    "flaws": {"type": "array", "items": {"type": "object"}},
+                    "trackers": {"type": "array", "items": {"type": "object"}},
+                    "affects": {"type": "array", "items": {"type": "object"}},
+                },
+            }
+        }
+    )
+    def get(self, request):
+        exploits = (
+            Exploit.objects.filter(source="CISA")
```

Comment:
We want all exploits irrespective of existing flaw, as reports need to tell if we have or do not have associated CVE. Also I am thinking if we want to filter to CISA here. Probably so for the time being.